library/Businessprocess/IcingaDbBackend.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Icinga\Module\Businessprocess;
+
+use Icinga\Module\Businessprocess\Common\IcingadbDatabase;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Icingadb\Model\Service;
+use ipl\Orm\Compat\FilterProcessor;
+use ipl\Orm\Query;
+
+class IcingaDbBackend
+{
+    use MonitoringRestrictions;
+
+    use IcingadbDatabase;
+
+    /** @var BpConfig */
+    protected $config;
+
+    protected $backend;
```

Comment:
Add PHPDoc and a newline after the member. It makes sense to rename this to `$conn` or `$db` though.